src/services/bonusService.ts

```diff
@@ -74,7 +74,7 @@ export default class BonusService {
     | IResponseSuccessAccepted
     | IResponseErrorForbiddenNotAuthorized
     | IResponseErrorGone
-    | IResponseErrorUnauthorized
+    | IResponseErrorUnauthorizedForLegalReasons
```

Comment:
`IResponseErrorUnauthorized` must stay since we return 401 in case the user is not authenticated